src/api-service/__app__/onefuzzlib/proxy.py

```diff
@@ -42,6 +42,7 @@
 # This isn't intended to ever be shared to the client, hence not being in
 # onefuzztypes
 class Proxy(ORMMixin):
+    Timestamp: Optional[datetime.datetime]
```

Comment:
Is there a straightforward way to avoid using a different casing convention in the Python code?